src/utils/db/db.utils.ts

```diff
@@ -62,3 +68,121 @@ export const logTeacher = async (
 
   return request.json();
 };
+
+export const getClass = async (
+  auth_token: string,
+  class_code: string
+): Promise<ClassPromise> => {
+  const options: RequestInit = {
+    headers: {
+      Authorization: `Bearer ${auth_token}`,
+    },
+  };
+
+  const request = await fetch(`${ENDPOINTS.CLASS}/${class_code}`, options);
+
+  return request.json();
+};
+
+export const joinClass = async (
+  auth_token: string,
+  class_id: string,
+  student_id: string
+): Promise<TypePromise<string>> => {
+  const bodyContent = {
+    class_id,
+    student_id,
+  };
+
+  const options: RequestInit = {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      Authorization: `Bearer ${auth_token}`,
+    },
+    body: JSON.stringify(bodyContent),
+  };
+
+  const request = await fetch(
+    `${ENDPOINTS.CLASS}/${class_id}/register`,
+    options
+  );
+
+  return request.json();
+};
+
+export type ClassInfo = {
+  subject: { id: string; name: string };
+  class_id: string;
+  start_time: string;
+  end_time: string;
+  finished_date: string;
+  LU: string;
+  MA: string;
+  MI: string;
+  JU: string;
+  VI: string;
+};
+
+export const createClass = async (
+  {
+    subject,
+    class_id,
+    start_time,
+    end_time,
+    finished_date,
+    LU,
+    MA,
+    MI,
+    JU,
+    VI,
+  }: ClassInfo,
+  teacher_id: string,
+  auth_token: string
+): Promise<TypePromise<string>> => {
+  const days = [];
```

Comment:
I feel like there was a better way to implement this using a map function. But this also helps readability